pkg/trace/api/otlp.go

```diff
@@ -318,9 +310,21 @@ func (o *OTLPReceiver) ReceiveResourceSpans(rspans ptrace.ResourceSpans, header
 	if env == "" {
 		env = o.conf.DefaultEnv
 	}
-	if !hostok {
+
+	// Get the hostname or set to empty
+	// if source is empty
```

Comment:
I don't think we need this comment nor the space (on the same note as my previous comment).